microstream/src/test/resources/logging.properties

```diff
@@ -0,0 +1,17 @@
+# Specify the handler, the handlers will be installed during VM startup.
```

Comment:
what is this file for? and why do we need it?

Reply:
Wasn't, it was a remnant of a failed attempt to debug postgres ;-)